internal/cfgplugins/mpls.go

```diff
@@ -53,3 +53,38 @@ func MPLSStaticLSP(t *testing.T, batch *gnmi.SetBatch, dut *ondatra.DUTDevice, l
 		gnmi.BatchReplace(batch, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Mpls().Config(), mplsCfg)
 	}
 }
+
+// Configure static MPLS label binding (LBL1) using CLI with deviation, if OC is unsupported on device
+func NewStaticMPLSLabel(t *testing.T, batch *gnmi.SetBatch, dut *ondatra.DUTDevice, lspName string, incomingLabel uint32, intfName string, nextHopIP string, protocolType string) {
```

Comment:
this looks like a near duplicate of `func MPLSStaticLSP`, please combine them.  (It's ok to rename the function to NewMPLSStaticLSP, but if you choose to do that, you'll need to update any dependencies.

Reply:
Yes, you'r correct but `func MPLSStaticLSP` is already merged into main and uses the duplicate deviation "StaticMplsLspOCUnsupported". If we combine `func NewMPLSStaticLSP`, the already merged code will result in an error. We also need to update the deviation to "StaticMplsUnsupported".

Please suggest what we can do?

Reply:
why would it fail?   The only difference in the code is the `if intfName != "" ` check.   Seems like that is compatible.

Reply:
No, I meant the deviation needs to be replaced from StaticMplsLspOCUnsupported to StaticMplsUnsupported. If I change it, the already merged code that checks for StaticMplsLspOCUnsupported won't find it and will go to the else part.

Reply:
Hi, what I think should be done here is instead of introducing `NewStaticMPLSLabel`, instead update the existing `MPLSStaticLSP` and use it in your test.

The only change is the  `if intfName != ""`  check which I think would be backwards compatible with how that function is currently. used.

Reply:
If you can successfully make this change, fix the static check issues and resolve conflicts I can approve quickly.